src/UI/Component/Modal/Lightbox.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace ILIAS\UI\Component\Modal;
+
+use ILIAS\UI\Component\Button;
+
+/**
+ * Interface Lightbox
+ *
+ * @package ILIAS\UI\Component\Modal
+ */
+interface Lightbox extends Modal
+{
+
```

Comment:
I think `LightboxPage` could be a valuable concept for the future. This could e.g. provide media type specific actions or download buttons (not being available for all items in the lightbox). `Panel` would be to general imo.